src/monitor/monitor.c

```diff
@@ -1789,18 +1789,14 @@ static errno_t monitor_config_file_fallback(TALLOC_CTX *parent_ctx,
     if (ret < 0) {
         err = errno;
         if (err == ENOENT) {
-             DEBUG(SSSDBG_MINOR_FAILURE,
-                   "file [%s] is missing. Will not update online status "
-                   "based on watching the file\n", file);
-             return EOK;
-
+            DEBUG(SSSDBG_CRIT_FAILURE,
+                  "file [%s] is missing. Will try again later.\n", file);
```

Comment:
Hi,

I think it is a good idea to start polling if the file is missing until it shows up. But I think there are valid use-cases where /etc/resolv.conf will never exists and in this case the polling might not be wanted. Would you mind to add a config option for the [sssd] section which controls where SSSD should expected that /etc/resolv.conf will show up eventually (new default) or it is expected that /etc/resolv.conf will not be available (current behavior)?

Reply:
Done, I have also changed the try_inotify documentation, as this setting will also affect future watched system configuration files.